Framework/DataObjects/src/MDBoxFlatTree.cpp

```diff
@@ -378,6 +379,90 @@ void MDBoxFlatTree::saveExperimentInfos(::NeXus::File *const file, const API::IM
   }
 }
 
+void MDBoxFlatTree::loadExperimentInfos(::NeXus::File *const file, const std::string &filename,
```

Comment:
Can we factor our portions of this member function that are the same as in `void MDBoxFlatTree::loadExperimentInfos(::NeXus::File *const file, const std::string &filename,
                                        const std::shared_ptr<Mantid::API::MultipleExperimentInfos> &mei, bool lazy)`?